src/components/Toggle/Toggle.module.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../styles/states";
+@import "../../styles/global-css-settings";
+
+/* in the new toggle we drew. */
+
+/* When the hidden checkbox will be focused by keyboard navigation events, the toggle appearance will reflect it */
+
+.input:focus-visible ~ .toggle,
+.input.focusVisible ~ .toggle {
```

Comment:
Maybe I not remember correctly but I think this :focus-visible class name is a global class we add for support visual focus in old browsers. Maybe it's not relevant anymore but please verify with @orrgottlieb .

Reply:
Good note, thanks!
Fixed